kalite/topic_tools/annotate.py

```diff
@@ -30,14 +30,15 @@ def update_content_availability(content_list, language="en"):
     # Loop through all content items and put thumbnail urls, content urls,
     # and subtitle urls on the content dictionary, and list all languages
     # that the content is available in.
+
+    # turn this whole function into a generator
     try:
         contents_folder = os.listdir(django_settings.CONTENT_ROOT)
     except OSError:
         contents_folder = []
 
     subtitle_langs = {}
-
-    updates = {}
+    updates = set()
```

Comment:
If we're just yielding an ```update``` object on every iteration, we probably don't want this here, as it will be making a reference to each yielded object and prevent them from being garbage collected.